media/css/mozorg/internet-health/privacy-security.scss

```diff
@@ -0,0 +1,616 @@
+        margin: 0;
+        padding: .825em 0 .825em 50px;
+        position: relative;
+
+        &:before {
+            @include font-size(20px);
+            background: $color-text-secondary;
+            border-radius: 100%;
+            color: #fff;
+            content: counter(li);
+            counter-increment: li;
+            display: block;
+            font-weight: bold;
+            height: 1.5em;
+            left: 0;
+            line-height: 1.55;
+            position: absolute;
+            text-align: center;
+            top: .65em;
+            width: 1.5em;
+        }
+
+        @media #{$mq-tablet} {
+            @include font-size(18px);
+            padding-left: 100px;
+
+            &:before {
+                left: 20px;
+            }
+        }
+    }
+}
+
+@media #{$mq-tablet} {
+    .topic-action-title {
+        padding: 20px 0 0 100px;
+        min-height: 60px;
+        position: relative;
+
+        &:before {
+            @include at2x('/media/img/mozorg/internet-health/privacy/action-sprite.png', 80px, 110px);
+            background-repeat: no-repeat;
+            content: '';
+            display: block;
+            left: 0;
+            position: absolute;
+            top: 10px;
+            width: 80px;
+            height: 55px;
+        }
+    }
+
+    .topic-action-moz .topic-action-title:before {
+        background-position: 0 0;
+    }
+
+    .topic-action-you .topic-action-title:before {
+        background-position: 0 -55px;
+    }
+}
+
+// User Control
+#control {
+    background: #c4e89c; // green
+
+    .topic-intro-aside:before {
+        @include at2x('/media/img/mozorg/internet-health/privacy/topic-control.png', 250px, 221px);
+    }
+
+    .pullquote {
+        background: #8fda80;
+    }
+}
+
+// Cybersecurity
+#security {
+    background: #cbf3f7; // blue
+
+    .topic-intro-aside:before {
+        @include at2x('/media/img/mozorg/internet-health/privacy/topic-security.png', 215px, 250px);
+    }
+
+    .pullquote {
+        background: #76f2ff;
+    }
+}
+
+// Government surveillance
+#surveillance {
+    background: #ffd8ee; // pink
+
+    .topic-intro-aside:before {
+        @include at2x('/media/img/mozorg/internet-health/privacy/topic-surveillance.png', 271px, 250px);
+    }
+
+    .pullquote {
+        background: #ff9ed5;
+    }
+}
+
+//*--------------------------------------------------------------------------*/
+// Health report
+
+.health-report {
+    @include font-size(20px);
+    background: #52fffe;
+    padding: 20px 0;
+
+    a:link,
+    a:visited {
+        color: $color-text-secondary;
+        display: inline-block;
+        font-weight: bold;
+    }
+
+    a:hover,
+    a:focus,
+    a:active {
+        color: $color-text-tertiary;
+    }
+
+    h2 {
+        margin-bottom: .75em;
+    }
+
+    .report-cta {
+        display: inline;
+        background: #feeb33;
+        padding: 0 20px;
+        -webkit-box-decoration-break: clone;
```

Comment:
I was thinking should we add a mixin for this? Seems like it might get used a fair bit on the new branding